lib/pxenforcer.js

```diff
@@ -182,28 +182,35 @@ class PxEnforcer {
     verifyUserScore(ctx, callback) {
```

Comment:
go deeper with that. you need to wrap your way down to the phin.

and wrap the p function as well